### DISCUSSION TITLE
Livewire 3 Livewire\Attributes\Rule must not be repeated

### DISCUSSION THREAD

**Discussion by LaurentOuattara** [livewire/livewire#6219] [1. Help] [answered]
I am learning livewire
I created a CreateProduct component in which the label is mandatory and minimum 4 characters
`#[Rule('required', message: 'Veuillez saisir un libellé svp', onUpdate: true)]
    #[Rule('min:4', message: 'Le titre doit avoir au moins 4 letttres')]
    public $libelle;`

But I get an error Livewire\Attributes\Rule must not be repeated
Help me please

**Comment by citricguy** [comment, marked as answer]
I think they fixed the `Livewire\Attributes\Rule must not be repeated` error in one of the recent updates, but there is still something going on with multiple rules with messages.

Try to update to the newest beta. I think the command is `composer update livewire/livewire`

**Reply by LaurentOuattara** [reply]
Thanks, it works after updating. But when I do my tests for the second min:4 rule it shows an error `Undefined array key "string"` . I do not know where it comes from

**Reply by citricguy** [reply]
I have one bug at https://github.com/livewire/livewire/discussions/6187 and there is another at https://github.com/livewire/livewire/discussions/6217 that I think are related.

**Comment by joshhanley** [comment]
@LaurentOuattara thanks for reporting! I suspect this issue is related to this failing test PR #6217.
